### PR TITLE
Define recommended conditions for policies

### PR DESCRIPTION
**Pull request by mmamczur**
**What type of PR is this?**
/kind feature
/kind gep
/kind documentation

**What this PR does / why we need it**:
Updates GEP-713 with recommended conditions and reasons that can be set in the policy status.
https://github.com/kubernetes-sigs/gateway-api/issues/1641

### CODE REVIEW

File: site-src/geps/gep-713.md [hunk 1]
```diff
@@ -331,6 +331,52 @@ Although these concerns are not unsolvable, they lead to the conclusion that
 a Kubectl plugin should be our primary approach to providing visibility here,
 with a possibility of adding policies to status at a later point.
 
+### Conditions
+To eanble easier standarized tool support the following set of conditions that
+the controllers can set in the policy status is recommended:
```
Comment (robscott) [inline site-src/geps/gep-713.md:336]:
```suggestion
Controllers using the Gateway API policy attachment model SHOULD populate the following
condition and reasons to provide a consistent experience across implementations.
```

**Review by robscott — COMMENTED** [review]
Thanks @mmamczur!

File: site-src/geps/gep-713.md (hunk 1 above)
Comment (mmamczur) [reply site-src/geps/gep-713.md]: updated with the suggestion

**Comment by robscott** [comment]
Thanks @mmamczur! This LGTM but will defer to @shaneutt or @youngnick for approval.

/lgtm

File: site-src/geps/gep-713.md [hunk 2]
```diff
@@ -331,6 +331,53 @@ Although these concerns are not unsolvable, they lead to the conclusion that
 a Kubectl plugin should be our primary approach to providing visibility here,
 with a possibility of adding policies to status at a later point.
 
+### Conditions
+Controllers using the Gateway API policy attachment model SHOULD populate the 
+following condition and reasons to provide a consistent experience across
```
Comment (sunjayBhatia) [inline site-src/geps/gep-713.md:336]: nit, tiny clarification on where this should go

```suggestion
following condition and reasons on policy resources to provide a consistent experience across
```

Comment (mmamczur) [reply site-src/geps/gep-713.md]: thanks, updated!

**Review by shaneutt — APPROVED** [review]

**Comment by robscott** [comment]
#1675 has merged so tests should pass again

/retest

**Pull request merged**